src/ctl.c

```diff
@@ -3083,3 +3085,76 @@ experimental_hooks_remove_ctl(tsd_t *tsd, const size_t *mib, size_t miblen,
 label_return:
 	return ret;
 }
+
```

Comment:
Suggestion: add some example of the return format / layout, something like. 

    /*
     * input[0]:  pointer_to_query  |  output[0]: extent_n_free_items
     *                              |  output[1]: ....
      ...

Also a simple description regarding what the 4 args do.